### DISCUSSION TITLE
New notifications badge still pops up when using NotificationsBlock

### DISCUSSION THREAD

**Discussion by alexevah** [AprilSylph/XKit-Rewritten#1068] [Help Desk] [answered]
New notifications badge still pops up even tho i blocked notifications for  certain post(s) using NotificationBlock, but when i click on them, there's nothing on my activity feed (as if i'm getting notifications from a shadowbanned blog)
posts i used the feature on:
https://www.tumblr.com/payidaresque/716331325887479808/fantastic-mr-fox-2009-dir-wes-anderson?source=share
https://www.tumblr.com/payidaresque/711159354012909568/joan-fontaine-letter-from?source=share

**Comment by marcustyphoon** [comment, marked as answer]
We can't fix this, unfortunately. (You can use Tweaks to hide the badge completely, which technically solves the problem, of course.)

But, fortunately, Tumblr is working on a version of this feature that's built into the site, and while I haven't tried it I assume that it will not have this issue! See https://www.tumblr.com/changes/716500434236342272/friday-may-5th-2023, and if you're a mobile user, try it out!

**Reply by alexevah** [reply]
Alright, thank you for reaching out so quickly! And thank you for your work.
